extensions/resource-deployment/src/ui/deployClusterWizard/deployClusterWizard.ts

```diff
@@ -175,4 +185,44 @@ export class DeployClusterWizard extends WizardBase<DeployClusterWizard, WizardP
 				throw new Error(`Unknown deployment type: ${type}`);
 		}
 	}
+
+	public async refreshPages() {
+		// All the providers will be handled differently
```

Comment:
This comment doesn't really provide any useful information. Ok great so they're handled differently...that's relevant to this particular function how?

Reply:
Unnecessary comment. Removing it